### DISCUSSION TITLE
Proposal: Refactor EmojiList to Use Unicode Emojis in Documentation Examples

### DISCUSSION THREAD

**Discussion by buraksakalli** [expo/expo#25342] [Proposals and Ideas]
Hello Expo Community,

I've been exploring the Expo documentation and came across the `EmojiList` component example. I noticed that the current implementation uses image assets for emojis. I'd like to propose a refactor of this component to use direct Unicode emojis instead.

### Proposed Changes
Here’s a snippet of the proposed refactoring:

```jsx

import { StyleSheet, FlatList, Text, Platform, Pressable } from 'react-native';

export default function EmojiList({ onSelect, onCloseModal }) {
  const emojis = ["😀", "😅", "😚", "🥰", "..."]; // More emojis can be added

  return (
    <FlatList
      horizontal
      showsHorizontalScrollIndicator={Platform.OS === 'web'}
      data={emojis}
      contentContainerStyle={styles.listContainer}
      renderItem={({ item, index }) => (
        <Pressable onPress={() => { onSelect(item); onCloseModal(); }}>
          <Text key={index} style={styles.emoji}>{item}</Text>
        </Pressable>
      )}
    />
  );
}

const styles = StyleSheet.create({
  listContainer: {
    // styles as before
  },
  emoji: {
    fontSize: 40, // Adjustable size
    marginRight: 20,
  },
});
```

Your feedback will be invaluable in ensuring that the proposed changes align well with the Expo's standards and community expectations.

Thank you for considering this proposal. I look forward to your insightful feedback and suggestions!

**Comment by tsapeta** [comment]
cc @amandeepmittal

**Comment by amandeepmittal** [comment]
Hey @buraksakalli, that's a great suggestion! To give you context for choosing images over unicodes, we plan to use the Expo Image library in the near future, so we opted for picking up emojis as images. But we'll definitely consider your proposal at the time we are going to update the tutorial and I will get back to you.

**Reply by buraksakalli** [reply]
Hi @amandeepmittal, Thanks for considering my suggestion. Excited to see the updates and am always here to support where needed.
